### PR TITLE
Add query strings to images in `osu!supporter`

### PR DESCRIPTION
**Pull request by geshii**
Fixing the images that did not update on the (production) wiki last time. As discussed a month ago, adding a random string to the images such as `?1` should fix the issue, so that's what I did.
I'm sorry that I couldn't fix this issue before, I was quite busy with IRL stuff.

### CODE REVIEW

**Comment by geshii** [comment]
There are build errors on stuff I didn't even edit. What should I do?

**Comment by geshii** [comment]
I might create an Issue linked to what just happened. These wiki pages are so old that I even have build errors while nothing is changed.

**Comment by TPGPL** [comment]
remark checks entire files for errors, not just the lines you have edited

**Comment by geshii** [comment]
> 
> 
> remark checks entire files for errors, not just the lines you have edited

I know, that's why creating an issue can be useful. These pages really need to be updated so people don't run into issues like that

File: wiki/osu!supporter/ja.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ osu!サポーターを入手するには、ログイン(または[登録](/wiki/
 
 買い物が終わったら、`カートに入れる`をクリックしてosu!storeのカートに入れます。
 
-![osu!storeカート](img/shopping-cart.jpg "osu!storeのカートにosu!suppoter for flyteを追加しました。")
+![osu!storeカート](img/shopping-cart.jp?1g "osu!storeのカートにosu!suppoter for flyteを追加しました。")
```
Comment (TPGPL) [inline wiki/osu!supporter/ja.md:35]:
```suggestion
![osu!storeカート](img/shopping-cart.jpg?1 "osu!storeのカートにosu!suppoter for flyteを追加しました。")
```

Comment (geshii) [reply wiki/osu!supporter/ja.md:35]: my bad

**Review by TPGPL — APPROVED** [review]

**Pull request merged**